src/components/dropdown/dropdown.ts

```diff
@@ -67,6 +75,9 @@ export class MDropdown extends BaseDropdown implements MDropdownInterface {
         if (text !== undefined && this.dirty && (this.internalFilterRegExp)) {
             result = this.internalFilterRegExp.test(text);
         }
+        if (this.internalItems.length < 4 && this.filterable) {
+            this.$refs.popup.$refs.popper.update();
```

Comment:
Ce code devrait plutôt aller à la fin de la méthode `buildItemsMap()`. la fonction `matchFilter` est utilisée par un dropdown-item pour savoir s'il est filtré ou non. Un peu comme un getter, ça ne devrait pas initier de comportements dans le composant autre que retourner true/false. Un item pourrait disparaître pour une autre raison que le search filtering (ex: v-if dont la valeur async)

Reply:
Peut-être pas nécessaire de mettre `this.filterable` dans la condition. En fait est-ce qu'on a vraiment besion d'une condition pour appeler `this.$refs.popup.$refs.popper.update()` à la fin du build?